CHANGELOG.md

```diff
@@ -80,11 +80,27 @@ All notable, unreleased changes to this project will be documented in this file.
 - Add --force flag to cleardb command - #5302 by @maarcingebala
 - Require non-empty message in orderAddNote mutation - #5316 by @maarcingebala
 - Stock management refactor - #5323 by @IKarbowiak
+- Update create permission group restriction - #5355 by @IKarbowiak
```

Comment:
This entire feature will be eventually merged from a single branch/PR and I think we don't need that granular changelog on `master`. I would suggest compiling all these messages and merging it with the one that we already have: `- Add permission groups - #5176, #5513 by @IKarbowiak`